test-sender/src/main.rs

```diff
@@ -71,28 +71,25 @@ async fn start_sender(config: TestSenderConfig) {
     let pubsub_topic = WakuPubSubTopic::from_str(pubsub_topic_str).unwrap();
     loop {
         for topic in config.topics.clone() {
+            let timestamp = Utc::now().timestamp();
```

Comment:
I think it would be better to generate the timestamp closer to building the messages, because it is more accurate to how time is generated in the radio implementation, and to avoid the middle steps like `gather_nodes` and `connect_multiaddresses`